### DISCUSSION TITLE
Até quantas requisições por minuto?

### DISCUSSION THREAD

**Discussion by resantosdosanjos** [bacen/pix-api#513] [General]
Aguenta até quantas requisições por minuto?

**Comment by arantesxyz** [comment]
É bem difícil responder essa questão, porque se tratam de vários fatores. Poderia ser relacionado ao PSP Pagador, ou ao próprio BCB ou ao PSP Recebedor, etc.

Eu diria que a resposta é "suficiente", independente de qual seja o seu projeto, acho que seja bem difícil atingir algum tipo de limite, sem ser aquele definido pelo PSP que te forneceu a API.

Caso queira saber sobre o arranjo por completo, o BCB disponibiliza publicamente algumas estatísticas do PIX em https://www.bcb.gov.br/estabilidadefinanceira/estatisticaspix

Se a gente considerar uma média de 120M de transações por dia, temos:

120M / dia
5M / hora
83.3K / minuto 
1.3k / segundo
